modules/module_05/code/Multiples.java

```diff
@@ -0,0 +1,24 @@
+//Objective: Print out the multiples of 7
+//Concepts: Initializing and indexing into arrays
+public class Multiples {
```

Comment:
I thought the intent of this example was a little ambiguous, so I would recommend using a simpler example at that point in the lecture, maybe with just Strings and no computation.